cosigned/base/kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: cosigned
+resources:
+- ../../cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/cosigned
+- ../../cluster-scope/base/rbac.authorization.k8s.io/clusterroles/cosigned
+- configmap.yaml
+- deployment.yaml
+- ../../cluster-scope/base/cosigned
```

Comment:
Same about this need to be on cluster side.
https://github.com/operate-first/apps/blob/d8add02ad6fd58a961ba404b6318c744ea255e19/cluster-scope/overlays/prod/moc/smaug/kustomization.yaml#L100